src/index.html

```diff
@@ -129,12 +146,63 @@ <h2 class="modal__title">New Post</h2>
     </div>
     <div class="modal" id="preview-modal">
       <div class="modal__content modal__content_type_preview">
-        <button aria-label="close modal" type="button" class="modal__close-button modal__close-button_type_preview"></button>
-        <img class="modal__image" src="./images/2-photo-by-ceiline-from-pexels.jpg" alt="image"/>
+        <button
+          aria-label="close modal"
+          type="button"
+          class="modal__close-button modal__close-button_type_preview"
+        ></button>
+        <img class="modal__image" src="#" alt="image" />
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script src="./scripts/validation.js"></script>
-    <script src="./scripts/index.js"></script>
+    <div class="modal modal_type_image">
+      <div class="modal__content">
+        <img class="modal__image" src="" alt="" />
+        <p class="modal__caption"></p> 
+        <button class="modal__close"></button>
+      </div>
+    </div>
```

Comment:
Please, delete the 2nd **image modal** from your `index.html`. You have it already in line 147